hbom/redis_backend.py

```diff
@@ -844,7 +843,7 @@ def prepare(cls, ref, pipe):
         definition = ref.__class__
         fields = getattr(definition, '_fields')
         s = cls.storage(_pk, pipe=pipe)
-        r = s.hmget(fields.keys())
+        r = s.hmget(list(fields.keys()))
```

Comment:
rather than forcing iterators into keys before passing to this function, wonder if it makes more sense to do it the other way from inside the function?

Reply:
hmm so making changes in redpipe instead? https://github.com/72squared/redpipe/blob/master/redpipe/keyspaces.py#L70